packages/identity/src/handle/index.ts

```diff
@@ -57,4 +59,39 @@ export class HandleResolver {
       return undefined
     }
   }
+
+  async resolveDnsBackup(handle: string): Promise<string | undefined> {
```

Comment:
I think that if any errors happen here that we want to `return undefined`, similar to the other resolver methods.

Reply:
ah yeah probs smart